docs/crud/measure/schema.md

```diff
@@ -0,0 +1,123 @@
+# CRUD Measures
+
+CRUD operations create, read, update and delete measures.
+
+[`bydbctl`](../../clients.md#command-line) is the command line tool in examples.
+
+## Create operation
+
+Create operation adds a new measure to the database's metadata registry repository. If the measure does not currently exist, create operation will create the schema.
+
+### Examples of creating
+
+A measure belongs to a unique group. We should create such a group with a catalog `CATALOG_MEASURE`
+before creating a measure.
+
+```shell
+$ bydbctl group create -f - <<EOF
+metadata:
+  name: sw_metric
+catalog: CATALOG_MEASURE
+resource_opts:
+  shard_num: 2
+  block_interval:
+    unit: UNIT_HOUR
+    num: 2
+  segment_interval:
+    unit: UNIT_DAY
+    num: 1
+  ttl:
+    unit: UNIT_DAY
+    num: 7
+EOF
+```
+
+The group creates two shards to store data points. Every day, it would create a
+segment that will generate a block every 2 hours.
+
+The data in this group will keep 7 days.
+
+Then, the below command will create a new measure:
+
+```shell
+$ bydbctl measure create -f - <<EOF
+metadata:
+  name: service_cpm_minute
+  group: sw_metric
+tag_families:
+- name: default
+  tags:
+  - name: id
+    type: TAG_TYPE_ID
+  - name: entity_id
+    type: TAG_TYPE_STRING
+fields:
+- name: total
+  field_type: FIELD_TYPE_INT
+  encoding_method: ENCODING_METHOD_GORILLA
+  compression_method: COMPRESSION_METHOD_ZSTD
```

Comment:
TBD, these options and explanations should be added in the concept chapters when you fix those docs.
Including how data layout in a KV system.

Reply:
Will add them in separate docs.